robot_main:
  ros__parameters:
    init_tcp_pose: [99.917 , -144.210 , 542.554 , -125.357 , 0.0 , -100.476]
    opencap_L_joint_pose: [-55.0, -90.0, -120.0, 30.0, 81.272, 0.0]
    opencap_R_joint_pose: [45.434, -124.551, 128.388, -184.270, 19.218,   0.0]
    desk_height: 177
    cola_height: 89
    cestbon_height: 83
    robot_speed: 0.5
    ball_R_joint_init_pose: [66.251, -123.403, 126.849, -184.040, -97.664, 90.0]
    ball_L_joint_init_pose: [-0.0, -66.994, -51.016, -87.069, 81.874, -90.0]
    ball_L_1_joint_pose: [-18.158, -127.475, -80.388, 28.213, 115.327, -90.0]
    ball_L_2_joint_pose: [-29.826, -136.002, -50.200, -3.902, 115.484, -89.971]
    ball_R_per_1_joint_pose: [108.784, -48.573, 65.323, -194.233, -129.820, 87.784]
    ball_R_1_joint_pose: [103.339, -23.279, 22.041, -180.788, -126.890, 90.0]
    ball_R_2_joint_pose: [93.138, -52.470, 69.132, -190.739, -123.491, 89.984]
